eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -378,6 +378,16 @@ page at http://checkstyle.sourceforge.net/config.html -->
     4) All classes should use ClientLogger as logger only but except ClientLogger itself -->
     <module name="com.azure.tools.checkstyle.checks.GoodLoggingCheck"/>
 
+    <!-- CUSTOM CHECKS -->
+    <!-- Public API classes should follow Camelcase rules: -->
+    <!-- 1. ClassName -->
+    <!-- 2. methodName -->
+    <!-- 3. variableName -->
+    <!-- 4. longerVariableOrMethodName -->
+    <!--<module name="com.azure.tools.checkstyle.checks.DisallowedWordsCheck">-->
+    <!--<property name="disallowedWords" value="URL, HTTP, XML, JSON, SAS, CPK" />-->
+    <!--</module>-->
```

Comment:
When you enable this, what issues do you see? If none, please do enable this rule!

Reply:
Will do.